tools/project_manager/remote_file_manager.php

```diff
@@ -1242,6 +1255,14 @@ function show_home_link()
     echo "<p><a href='?action=showdir'>$text</a></p>\n";
 }
 
+function show_commons_link()
+{
+    global $commons_rel_dir;
+    $text = sprintf(_("Return to Commons folder"));
+    $url = "?cdrp=" . urlencode($commons_rel_dir);
```

Comment:
I thought this was going to be changed to "Go to Commons folder", since if their home directory isn't the Commons, "Return" doesn't make sense.

Reply:
This was changed, I believe you were looking at an older commit.